### DISCUSSION TITLE
Invalid Audience : Get User Info

### DISCUSSION THREAD

**Discussion by ranjiteds** [zitadel/zitadel#6063] [Q&A] [answered]
Hi,

I am getting invalid audience error using any user related api calls 

url: 'https://' + domain + '/auth/v1/users/me'

These is my request with access token and access token is valid as I am able to use introspect call with success.

const headers = {
    'Accept': 'application/json',
    'Authorization': 'Bearer ' + accessToken
}

let config = {
        method: 'get',
        url: 'https://' + domain + '/auth/v1/users/me',
        headers
  ;

**Comment by mffap** [comment, marked as answer]
Hi @ranjiteds can you please check if the information in the troubleshooting guide helps? 
https://zitadel.com/docs/support/troubleshooting#invalid-audience

Probably need to add the correct scope to access the zitadel apis.
